### ISSUE TITLE
Rendering of markdown table

### ISSUE THREAD

**Issue by sschob** [jkitchin/ox-ipynb#12]
Hi, 
first of all, this is great work. I encountered the following issue when exporting tables. The second line (after the header) results in "|--". While this is fine at least with Jupyter 5.5, this is not correctly rendered in Jupyter 5.7. 

For example this org table
```
| h1 | h2 |
| --- + --- |
|c1  | c2 |
```
becomes
```
|Header1|Header 2|
|---
|content1|content2|
```
but should be 
```
|Header1|Header 2|
|---|---|
|content1|content2|
´´`

**Comment by jkitchin** [comment]
Thanks for reporting this. Seems weird 5.7 changes how it renders markdown. I pushed a fix that should work for 5.7, and also seems to work in 5.5. Let me know if it works for you.

**Comment by sschob** [comment]
Thanks for fixing this! Works for me too.
